numpy/lib/tests/test_function_base.py

```diff
@@ -2556,6 +2561,17 @@ def test_0d_comparison(self):
         y = piecewise(x, [x <= 3, (x > 3) * (x <= 5), x > 5], [1, 2, 3])
         assert_array_equal(y, 2)
 
+        assert_raises_regex(ValueError, '2 or 3 functions are expected',
+            piecewise, x, x, [x <= 3, x > 3], [1])
+        assert_raises_regex(ValueError, '2 or 3 functions are expected',
+            piecewise, x, x, [x <= 3, x > 3], [1, 1, 1, 1])
```

Comment:
Typos here - should be `x` just once. Alarming that only some platforms are failing...